packages/avatars/stories/examples/Default.tsx

```diff
@@ -0,0 +1,232 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useMemo } from 'react';
+import styled from 'styled-components';
+import { Story } from '@storybook/react';
+import { LG } from '@zendeskgarden/react-typography';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import UserIcon from '@zendeskgarden/svg-icons/src/16/user-solo-stroke.svg';
+import ZendeskIcon from '@zendeskgarden/svg-icons/src/26/zendesk.svg';
+import { PALETTE } from '@zendeskgarden/react-theming';
+
+const StyledTable = styled.table.attrs<Partial<IStoryProps>>(p => ({
+  style: { backgroundColor: p.surfaceColor }
+}))<Partial<IStoryProps>>`
+  width: 100%;
+  border-spacing: 0;
+`;
+
+const StyledAvatarTd = styled.td`
+  padding: ${p => p.theme.space.lg} 0;
+  text-align: center;
+`;
+
+interface IStoryProps {
+  size: 'extraextrasmall' | 'extrasmall' | 'small' | 'medium' | 'large';
+  status: 'none' | 'away' | 'available';
```

Comment:
Will the lack of `status` animation transitions be corrected when this PR is merged with @hzhu's fix? https://github.com/zendeskgarden/react-components/pull/889/commits/fc68647ba29ef28d5f57be618c82cd44f7a58f82

Reply:
Is this not showing animations for you? I've already merged `main` into this branch and the animations are working well for me.

Reply:
Oh, interesting. They are animating as expected on the Canvas tab, but not on the Docs tab. See https://5f84ace4f41bdb249d521b09--zendeskgarden-react-components.netlify.app/storybook/?path=/docs/components-avatars--default